bessctl/conf/samples/arp.bess

```diff
@@ -0,0 +1,11 @@
+import scapy.all as scapy
+
+eth_header = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='ff:ff:ff:ff:ff:ff')
+arp_header = scapy.ARP(op=1, pdst='1.1.1.1')
```

Comment:
Maybe we can use another IP address that is not symmetric? Just as a safety measure for subtle endianness regression bugs later any time...